### DISCUSSION TITLE
Is there an equivalent to opensea-js's transfer?

### DISCUSSION THREAD

**Discussion by Chinch-Bug** [ProjectOpenSea/seaport-js#577] [Q&A] [answered]
I saw that the opensea-js sdk has a method for transfering tokens between wallets. Does seaport-js have any equivalent for transfering NFTs between wallets? (I assume not, but I figured I might've missed it while looking)
Wanted to know if it'd be possible to estimate the gas needed for the transfer

**Comment by ryanio** [comment, marked as answer]
You can just copy over the method to your own app code if you want to use it without OpenSea-js, it just uses ethers.js
